src/User.php

```diff
@@ -34,7 +36,8 @@ public static function fromVerifier(Verifier $verifier): self
 			(string)$verifier->getAttribute('CompanySSN'),
 			(string)$verifier->getAttribute('CompanyName'),
 			(string)$verifier->getAttribute('DestinationSSN'),
-			(string)$verifier->getAttribute('AuthID')
+			(string)$verifier->getAttribute('AuthID'),
+			(string)$verifier->getAttribute('Mobile'),
```

Comment:
Remove the trailing `,` since we support php before 7.3

Reply:
Done, thank you @sunkan